packages/plugin-ext/src/main/node/paths/plugin-paths-service.ts

```diff
@@ -17,13 +17,17 @@
 import { injectable, inject } from 'inversify';
 import { FileSystem, FileStat } from '@theia/filesystem/lib/common';
 import * as path from 'path';
+import { readdir, remove} from 'fs-extra';
```

Comment:
missing whitespace, i though we have format on save enabled. which ide do you use?

Reply:
I use Jetbrains products, we are really wasting everybody's time by not having a CLI level formatting solution...

I can see if I have time next week to make a PR with such a configuration, however it will re-format the entire project forcing many re-bases which means somebody from the core team will have to take it farther... (I can make a "pure" PR with the configuration only...)

Are you interested in that?

Reply:
I fixed the WS